src/annotator/test/sidebar-test.js

```diff
@@ -154,15 +154,26 @@ describe('Sidebar', () => {
     fakeDragHandler = {
       destroy: sinon.stub(),
     };
-    FakeDragHandler = sinon.stub().returns(fakeDragHandler);
```

Comment:
With the changes in these mocks, the existing tests fail without the changes in sidebar.tsx.